src/components/pages/Dashboard/Dashboard.js

```diff
@@ -75,7 +75,9 @@ const Dashboard = () => {
         leftElement={Title}
         rightElement={
           <Button color="teal" fixed="right">
-            <Link style={{color: 'white'}} to="/addfarmer">Add Farmer</Link>
+            <Link style={{ color: 'white' }} to="/addfarmer">
+              Add Farmer
+            </Link>
           </Button>
         }
```

Comment:
So from line 75 to line 82 can be deleted because PageHeader changed now. I would suggest adding a 'Add Farmer' button next to 'Settings' in the nav bar 🔥 There is no way to add farmers if this is not done as the previously displayed button doesn't appear anymore.